lib/charms/ovn_charm.py

```diff
@@ -102,13 +103,8 @@ class OVNChassisCharmRelationAdapters(
 class BaseOVNChassisCharm(charms_openstack.charm.OpenStackCharm):
     """Base class for the OVN Chassis charms."""
     abstract_class = True
-    package_codenames = {
-        'ovn-host': collections.OrderedDict([
-            ('2', 'train'),
-            ('20', 'ussuri'),
-        ]),
-    }
-    release_pkg = 'ovn-host'
+    package_codenames = os_utils.PACKAGE_CODENAMES
+    release_pkg = 'neutron-common'
```

Comment:
Do you know that the `ovn-chassis` subordinate will be deployed together with something that installs the `neutron-common` package?

Does the `ovn-dedicated-chassis` principal charm install this package?

Reply:
> Do you know that the `ovn-chassis` subordinate will be deployed together with something that installs the `neutron-common` package?

No you're right, ovn-chassis's readme says it's a subordinate for ovn-central but neutron-common isn't a dependency of ovn-central. I need to figure out something else, thanks!